tests/e2e/tests/critical-path/browser/search-capabilities.e2e.ts

```diff
@@ -0,0 +1,134 @@
+        keyName = common.generateWord(10);
+        await browserPage.addHashKey(keyName);
+    })
+    .after(async() => {
+        // Clear and delete database
+        await browserPage.deleteKeyByName(keyName);
+        await cliPage.sendCommandsInCli([`DEL ${keyNames.join(' ')}`, `FT.DROPINDEX ${indexName}`]);
+        await deleteStandaloneDatabaseApi(ossStandaloneConfig);
+    })('RediSearch capabilities in Browser view to search per Hashes or JSONs', async t => {
+        indexName = `idx:${keyName}`;
+        keyNames = [`${keyName}:1`, `${keyName}:2`, `${keyName}:3`];
+        const commands = [
+            `HSET ${keyNames[0]} "name" "Hall School" "description" " Spanning 10 states" "class" "independent" "type" "traditional" "address_city" "London" "address_street" "Manor Street" "students" 342 "location" "51.445417, -0.258352"`,
+            `HSET ${keyNames[1]} "name" "Garden School" "description" "Garden School is a new outdoor" "class" "state" "type" "forest; montessori;" "address_city" "London" "address_street" "Gordon Street" "students" 1452 "location" "51.402926, -0.321523"`,
+            `HSET ${keyNames[2]} "name" "Gillford School" "description" "Gillford School is a centre" "class" "private" "type" "democratic; waldorf" "address_city" "Goudhurst" "address_street" "Goudhurst" "students" 721 "location" "51.112685, 0.451076"`,
+            `FT.CREATE ${indexName} ON HASH PREFIX 1 "${keyName}:" SCHEMA name TEXT NOSTEM description TEXT class TAG type TAG SEPARATOR ";" address_city AS city TAG address_street AS address TEXT NOSTEM students NUMERIC SORTABLE location GEO`
+        ];
+
+        // Create 3 keys and index
+        await cliPage.sendCommandsInCli(commands);
+        // Verify that user see the tooltips for the controls to switch the modes
+        await t.click(browserPage.patternModeBtn);
+        await t.hover(browserPage.patternModeBtn);
+        await t.expect(browserPage.tooltip.textContent).contains(patternModeTooltipText, 'Invalid text in pattern mode tooltip');
+        await t.hover(browserPage.redisearchModeBtn);
+        await t.expect(browserPage.tooltip.textContent).contains(redisearchModeTooltipText, 'Invalid text in redisearch mode tooltip');
+
+        // Verify that user see the "Select an index" message when he switch to Search
+        await t.click(browserPage.redisearchModeBtn);
+        await t.expect(browserPage.keyListTable.textContent).contains(notSelectedIndexText, 'Select an index message not displayed');
+
+        // Verify that user can search by index in Browser view
+        await browserPage.selectIndexByName(indexName);
+        await verifyKeysDisplayedInTheList(keyNames);
+        await t.expect((await browserPage.getKeySelectorByName(keyName)).exists).notOk('Key without index displayed after search');
+        // Verify that user can search by index plus key value
+        await browserPage.searchByKeyName('Hall School');
```

Comment:
You create am index with different indexed fields by Text, Numeric, Tag. 
Can you please add to test more complex check like search not only by key value but for description + name + type + city or something else?

Reply:
thanks, added